controllers/util/drpolicy_util.go

```diff
@@ -113,3 +113,13 @@ func GetSecondsFromSchedulingInterval(drpolicy *rmn.DRPolicy) (float64, error) {
 		return s.Seconds(), err
 	}
 }
+
+func DrpolicyContainsDrcluster(drpolicy *rmn.DRPolicy, drcluster string) bool {
```

Comment:
Ideally, I would want DR in the names to be uppercase but we can make that change later.